File: exercise1/project_c/projec1/Form1.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
namespace projec1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        { 
            if(!String.IsNullOrEmpty(textBox1.Text))
            {
                MessageBox.Show("successfull");
            }else
            {
                textBox1.BackColor = Color.Red;
            }
           
        }

        private void button2_Click(object sender, EventArgs e)
        {
            this.Close();
            
        }

        private void textBox1_TextChanged(object sender, EventArgs e)
        {

            
        }

        private void label1_Click(object sender, EventArgs e)
        {

        }
    }
}

File: exercise1/project_vb/project1_vb/Form1.vb
Imports System.Windows.Forms.VisualStyles.VisualStyleElement

Namespace projec1
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs)
            If Not String.IsNullOrEmpty(textBox1.Text) Then
                MessageBox.Show("successfull")
            Else
                textBox1.BackColor = Color.Red
            End If
        End Sub

        Private Sub button2_Click(ByVal sender As Object, ByVal e As EventArgs)
            Me.Close()
        End Sub

        Private Sub textBox1_TextChanged(ByVal sender As Object, ByVal e As EventArgs)
        End Sub

        Private Sub label1_Click(ByVal sender As Object, ByVal e As EventArgs)
        End Sub
    End Class
End Namespace
